hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -334,15 +329,11 @@ public boolean setVolumeOwner(String volumeName, String owner)
   }
 
   @Override
-  public void setVolumeQuota(String volumeName, long quotaInCounts,
-      long quotaInBytes) throws IOException {
+  public void setVolumeQuota(String volumeName, long quotaInBytes,
+      long quotaInCounts) throws IOException {
```

Comment:
Can we align the  quotaInBytes and quotaInCounts order in setVolumeQuota and setBucketQuota？  In another words, no need for the code change here.